### DISCUSSION TITLE
Init driver before start

### DISCUSSION THREAD

**Discussion by AndrejSayadov** [seleniumbase/SeleniumBase#973] [Q&A] [answered]
Hello! Can I immediately redefine driver before starting? Now I'm stopping the old driver and connecting a new one to it. I want to immediately specify the necessary driver.
<img width="728" alt="image" src="https://user-images.githubusercontent.com/89129698/130804748-fe43fb88-c165-475c-8a88-1c9ec889bb26.png">

**Comment by mdmintz** [comment, marked as answer]
You should always use ``self.driver.quit()`` instead of ``self.driver.close()``, because otherwise you might leave open running processes. That being said, running the above code from your example will lose all the SeleniumBase settings that are set when the browser is launched. In your particular case, you may want to fork/clone SeleniumBase and then go into ``seleniumbase/core/browser_launcher.py`` and replace ``from selenium import webdriver`` with ``from seleniumwire import webdriver`` so that you can run your specific seleniumwire code. But if you don't care about the SeleniumBase default options, then you can probably do what you're trying to do now if you include ``from seleniumwire import webdriver`` in your example and then run the code you had.
